### DISCUSSION TITLE
2022-05-13 Explorer Team Updates

### DISCUSSION THREAD

**Discussion by frol** [near/community#273] [Pagoda Platform]
Explorer team update is here! 📻 

We have decided our path forward with the design code! NEAR Explorer will receive Pagoda design.

NFT and FT assets tabs design is still in the works, and we have a few early mockups:

<img width="770" alt="image" src="https://user-images.githubusercontent.com/304265/168669963-5bcb716a-7fba-491c-b2b2-43e0dae13fe9.png">

<img width="770" alt="image" src="https://user-images.githubusercontent.com/304265/168670071-82f460c2-828f-4d47-8be2-d47ceab2431c.png">

Transaction Details page got another round of nice touches:

<img width="770" alt="image" src="https://user-images.githubusercontent.com/304265/168670675-bcc2ef01-637d-4227-9d85-06a3efd9a417.png">

There has also been a few nice improvements on the technical implementation side of Explorer backend which you can find here:

https://github.com/near/near-explorer/compare/5ff970a2213cdffec4c7e9ff6beb3543289de638...163742a91ea4ebd04ccbf421dd081744455a74d6

----

If you have any questions, feel free to drop us a message here!

**Comment by camchis** [comment]
This looks fucking awesome
